fizzbuzz.js

```diff
@@ -1,12 +1,21 @@
+const Error = require('./statusCodes.js');
```

Comment:
In JavaScript, we only capitalize variable names when that variable contains a constructor/class